dist/datasource.js

```diff
@@ -219,13 +234,17 @@ System.register(['lodash', './showdown.min.js', './query_builder'], function (_e
         }, {
           key: 'query',
           value: function query(options) {
+            
             var from = Math.floor(options.range.from.valueOf() / 1000);
             var to = Math.floor(options.range.to.valueOf() / 1000);
 
             var targets = options.targets.filter(function (t) {
               return !t.hide;
             });
 
+            console.log("In datasource.js, showing targets\n");
+            console.log(targets);
```

Comment:
Please ignore the `console.log()` statements. These are just helping me with the request/response objects.